gpiozero/boards.py

```diff
@@ -1686,3 +1687,52 @@ def off(self):
         Turns the socket off.
         """
         self.value = False
+
+
+class JamHat(CompositeOutputDevice):
+    """
+    Extends :class: `CompositeOutputDevice` for the `ModMyPi JamHat` board.
+
+    There are 6 LEDs, two buttons and a tonal buzzer. The pins are fixed.
+
+    Usage:
+        from gpiozero import JamHat
+
+        hat = JamHat()
+        hat.lights_1.on()
+        hat.buzzer.play(2000)
+        hat.off()
+
+    :param bool pwm:
+    If ``True``, construct :class: PWMLED instances to represent each LED 
+    on the board. If ``False`` (the default), construct regular :class: 
+    LED instances.
+
+    :param bool initial_value:
+    If ``False`` (the default), all LEDs will be off initially. If
+    ``None``, each device will be left in whatever state the pin is found
+    in when configured for output (warning: this can be on). If ``True``,
+    the device will be switched on initially.
```

Comment:
Should the initial_value also affect the buzzer?

Reply:
I guess it should (despite my increasing hatred of the gurgly buzzers being on at all :laughing:)

Reply:
But to what note?

Reply:
Given that a Servo has range -1 to +1, and has an initial value of 0, perhaps the default initial value of a TonalBuzzer that ranges from 0 to 1 should be 0.5 ?

Reply:
`TrafficHat` doesn't have an initial value...

Reply:
True - TrafficLightsBuzzer doesn't have an `initial_value` so maybe JamHat shouldn't either.

Reply:
OK, so I'm going to leave this and not add an `initial_value` for JamHat.

Reply:
I think that's a good call. I wonder if we should remove `initial_value` from composite devices other than `LEDBoard`. a) it gets complicated b) we only really turn LEDs on c) we waste time discussing what it should do every time

Reply:
Which classes (other than LEDBoards and LEDCollections) have `initial_value`?

Reply:
Hmmm, looks like you're right - I must have been getting confused with something else, sorry!